tests/integration-tests/tests/scheduler_plugin/test_scheduler_plugin.py

```diff
@@ -533,3 +535,54 @@ def _test_cluster_update(cluster, cluster_config):
     cluster.update(str(cluster_config), force_update="true")
     cluster.start()
     _check_fleet_status(cluster, "RUNNING")
+
+
+def _test_invoke_scheduler_plugin_event_handler_script(command_executor, compute_node, run_as_user):
+    """Test invoke-scheduler-plugin-event-handler.sh."""
+    # Create a dummy cluster config for test, echo "dummy config {Event} executed" during handler execution.
+    cluster_config = "/opt/parallelcluster/shared/cluster-config.yaml"
+    dummy_cluster_config = "/opt/parallelcluster/shared/dummy-cluster-config.yaml"
+    command_executor.run_remote_command(
+        f"sudo sed 's/echo \\\\\"/echo \\\\\"dummy config /g' {cluster_config} | sudo tee "
+        f"{dummy_cluster_config} >/dev/null"
+    )
+    command_executor.run_remote_command(f"sudo sed -i 's/echo \"/echo \"dummy config /g' {dummy_cluster_config}")
```

Comment:
what this second sed for?

Reply:
The second sed is for replacing echo in:
```
HeadClusterUpdate:
  ExecuteCommand:
    Command: env && echo "dummy config HeadClusterUpdate executed"
```
The first sed is for replacing echo in:
```
HeadComputeFleetUpdate:
  ExecuteCommand:
    Command: "env\nSTATUS=$(cat ${PCLUSTER_COMPUTEFLEET_STATUS} | jq -r '.status')\n\
      if [[ \"${STATUS}\" == \"STOP_REQUESTED\" ]] && [[ ! -f \"stop_failure\"\
      \ ]]; then\n  echo \"HeadComputeFleetUpdate failing the first stop execution\"\
      \n  touch stop_failure && exit 1\nfi\nif [[ \"${STATUS}\" == \"STOP_REQUESTED\"\
      \ ]] && [[ ! -f \"stop_executed\" ]]; then\n  echo \"HeadComputeFleetUpdate\
      \ stop executed\"\n  touch stop_executed && exit 0\nfi\nif [[ \"${STATUS}\"\
      \ == \"START_REQUESTED\" ]] && [[ ! -f \"start_failure\" ]]; then\n\
      \  echo \"HeadComputeFleetUpdate failing the first start execution\"\
      \n  touch start_failure && exit 1\nfi\nif [[ \"${STATUS}\" == \"START_REQUESTED\"\
      \ ]] && [[ ! -f \"start_executed\" ]]; then\n  echo \"HeadComputeFleetUpdate\
      \ start executed\"\n  touch start_executed && exit 0\nfi\necho \"dummy config HeadComputeFleetUpdate\
      \ not supposed to be here\"\ntouch update_wrong_execution\n"
```